charts/seed-bootstrap/charts/fluent-bit/values.yaml

```diff
@@ -33,3 +33,6 @@ lokiLabels:
     hostname: "host_name"
     unit: "systemd_component"
 exposedComponentsTagPrefix: "user-exposed"
+
```

Comment:
This value wouldn't be needed anymore if you move the event-logger to a sub package as mentioned before.